schemas/continuity.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+const schemas = require('bedrock-validation').schemas;
+
+const signature = {
```

Comment:
This exists:

https://github.com/digitalbazaar/bedrock-validation/blob/master/schemas/linkedDataSignature.js

Can we just reuse it? Maybe it needs an option for restricting the linked data signature type to just `LinkedDataSignature2015`. We could potentially just add a TODO to reuse it if that gets us there faster -- and can address later.